File: RB101/study_guide/live_assesment/small_problems/easy_2/retire.rb
puts "What is your age?"
age = gets.chomp.to_i

puts "At what age would you like to retire?"
retire_at = gets.chomp.to_i

year = Time.now.year

retire_year =  year + (retire_at - age)
to_go = retire_at - age

puts "Its #{year}. You will retire in #{retire_year}"
puts "You only have #{to_go} years left to go!"

File: JS210/exersies/easy3/retire.js
const readline = require('readline-sync');
let today = new Date();
let year = today.getFullYear();

let age = readline.question('What is your age? ');
let retireAt = readline.question('What age would you like to retire? ');

console.log(`It's ${year}. You will retire in ${(retireAt - age) + year}.`);
console.log(`You only have ${retireAt - age} to go!`);



